api/catalog/v1alpha1/shared_types.go

```diff
@@ -53,6 +56,17 @@ const (
 	BindingConditionReasonSecretAccessRequestExpired    = "SecretAccessRequestExpired"
 	BindingConditionReasonSecretAccessRequestApproved   = "SecretAccessRequestApproved"
 	BindingConditionReasonSecretAccessRequestDenied     = "SecretAccessRequestDenied"
+
+	BindingConditionReasonGatewayNotAccepted          = "GatewayNotAccepted"
+	BindingConditionReasonGatewayNotProgrammed        = "GatewayNotProgrammed"
+	BindingConditionReasonListenerNotAccepted         = "ListenerNotAccepted"
+	BindingConditionReasonListenerNotProgrammed       = "ListenerNotProgrammed"
+	BindingConditionReasonListenerUnresolvedReference = "ListenerUnresolvedReference"
+	BindingConditionReasonRouteNotAccepted            = "RouteNotAccepted"
+	BindingConditionReasonUnresolvedBackendReference  = "UnresolvedBackendReference"
```

Comment:
`BackendUnresolvedReference`.

All the reasons are of structture `<item><issue>`